kube/charts/kubernetes/kube.go

```diff
@@ -33,6 +34,12 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+const (
+	headlessPort      = 55555
```

Comment:
any reason using this specific value?

Reply:
taken from compose-on-kube, no specific reason I think